### PR TITLE
User object

### PR DESCRIPTION
**Pull request by dsloan92**
Created the User Object so that it can be utilized with the login page. Should be a quick review... Hopefully.

### CODE REVIEW

File: classes/user.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+class User
+{
+    private $userIsAdmin;
+    private $userID;
+    private $_db;
+
+    /**
+     * User constructor.
+     * @param $userEmail email of user logging into system
+     * @param $_db database object
```
Comment (kellerflint) [inline classes/user.php]: @param requires datatype

File: model/Database.php [hunk 2]
```diff
@@ -345,5 +345,20 @@ function importUser($user_id) {
 
         $statement->execute([$user_id, $year]);
     }
+    /**
+     * Get user by it's email
+     * @param int $user_email users email
+     * @return a row of information for user
```
Comment (kellerflint) [inline model/Database.php]: @return requires datatype (I believe it would be an array for this?)

File: classes/user.php [hunk 3]
```diff
@@ -0,0 +1,51 @@
+<?php
```
Comment (kellerflint) [inline classes/user.php]: I believe the names of class files are capitalized (User.php)

File: classes/user.php (hunk 1 above)
Comment (dsloan92) [reply classes/user.php]: Updated both @param with a datatype

File: model/Database.php (hunk 2 above)
Comment (dsloan92) [reply model/Database.php]: Updated the datatype for @return

File: classes/user.php (hunk 3 above)
Comment (dsloan92) [reply classes/user.php]: Updated!

**Review by kellerflint — APPROVED** [review]
Awesome code! Merging...

**Pull request merged**